addon.py

```diff
@@ -299,9 +331,13 @@ def _make_nfo_files( self, film, episode, dirname, filename, videourl ):
 			with closing( xbmcvfs.File( filename, 'w' ) ) as file:
 				file.write( b'<episodedetails>\n' )
 				file.write( bytearray( '\t<title>{}</title>\n'.format( film.title ), 'utf-8' ) )
-				file.write( b'\t<season>1</season>\n' )
-				file.write( b'\t<autonumber>1</autonumber>\n' )
-				file.write( bytearray( '\t<episode>{}</episode>\n'.format( episode ), 'utf-8' ) )
+				if nfomode == 2 and season is not None and episode is not None:
+					file.write( bytearray( '\t<season>{}</season>\n'.format( season ), 'utf-8' ) )
+					file.write( bytearray( '\t<episode>{}</episode>\n'.format( episode ), 'utf-8' ) )
+				elif nfomode == 2:
+					file.write( b'\t<season>999</season>\n' )
+					file.write( bytearray( '\t<episode>{}</episode>\n'.format( sequence ), 'utf-8' ) )
```

Comment:
Doesn't that transition from mostly broken to always broken?
Use-case: Users download a lot, never look at NFOs and have the same problems as before,
Additionally, users who do download in temporal sequence still get borked NFOs.
This just makes sure that no auto-NFO can ever be used without editing.

Reply:
To be considered: reordering episodes requires that KODI's database needs to be updated too. +1 for non-trivial.

Reply:
You say: _"Use-case: Users download a lot, never look at NFOs and have the same problems as before"_

That's right. But in any case, Season 999 does not conflict with real season/episode data. And in case the user does not look and manage NFOs: 999=1=666=761=672=....

Right?

Benefit: everything else that season=[really used season numbers] does not conflict with correct filenames and NFOs

Reply:
Final result: we transition from mostly broken to sometimes broken